hw14/src/test/java/base/Hw14.java

```diff
@@ -0,0 +1,77 @@
+package base;
+
+import org.openqa.selenium.Beta;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.List;
+
+
+public class Hw14 extends BaseTest {
+    List<String> files = Files.readAllLines(Paths.get(this.getClass().getClassLoader().getResource("LogDate.properties").toURI()));
+
+    public Hw14() throws IOException, URISyntaxException {

```

Comment:
why do you need default constructor here? it is available by default in each class